stories/toaster.stories.tsx

```diff
@@ -58,7 +58,43 @@ storiesOf('Toaster', module)
       <Button
         color="primary"
         onClick={() => {
-          Toast('error', 'This is a toaster!', 'Error')
+          Toast('error', 'This is a toaster!', 'bottom-right', 'Error and position is bottom-right')
+        }}
+      >
+        Click me!
+      </Button>
+      <Toaster autoClose={3000} hideProgressBar draggable />
+    </div>
+  ))
+  .add('Default', () => (
+    <div>
+      <Button
+        color="defult"
+        onClick={() => {
+          Toast(
+            'default',
+            'This is a toaster!',
+            'bottom-left',
+            'Default and position is bottom-left',
```

Comment:
Default position will be top right. 

To properly show the default behavior, the call to our toast function should look like: 

```
Toast(
            'default',
            'This is a toaster!',
            'Default and position is top-right',
)
```